NeuroID/src/main/java/com/neuroid/tracker/NeuroID.kt

```diff
@@ -188,7 +188,13 @@ class NeuroID
         data class Builder(val application: Application? = null,
                            val clientKey: String = "",
                            val serverEnvironment: String = PRODUCTION) {
-            fun build() = NeuroID(application, clientKey, serverEnvironment)
+            fun build() {
+                if (singleton == null) {
+                    val neuroID = NeuroID(application, clientKey, serverEnvironment)
+                    neuroID.setupCallbacks()
+                    setNeuroIDInstance(neuroID)
```

Comment:
Does this mean we need to update our documentation so that they don't call `setNeuroIDInstance` right after build anymore?

Reply:
yes. that is correct.